week_02/case_study_02.R

```diff
@@ -1 +1,50 @@
- 
\ No newline at end of file
+## Zaque Evans - CS02 - GEO 511
+## due Sep_17_2020
+
+## install if needed (do this exactly once):
+install.packages("usethis")
```

Comment:
better to not include this in your script - just run it on the command line and then use `library()` in the script.